app/src/main/java/com/locoquest/app/MainActivity.kt

```diff
@@ -21,14 +21,16 @@ import com.google.firebase.auth.ktx.auth
 import com.google.firebase.ktx.Firebase
 
 
+private val TAG : String = MainActivity::class.java.name
+private lateinit var auth: FirebaseAuth
+private const val REQ_ONE_TAP = 0
+private lateinit var oneTapClient: SignInClient
+private lateinit var signUpRequest: BeginSignInRequest
+private lateinit var signInButton: SignInButton
+private var showOneTapUI = true
+
 class MainActivity : AppCompatActivity(), OnMapReadyCallback{
```

Comment:
REQ_ONE_TAP needs declaration as a constant and the variables have to be declared the top level

Reply:
The variables are OK where they were: within the Activity class.